docs/reference/relnotes/_index.md

```diff
@@ -0,0 +1,11 @@
+---
+title: "Knative Release Notes"
+linkTitle: "Release notes"
+weight: 10
+type: "docs"
+---
+
+- For details about the latest Knative Serving release, see the [https://github.com/knative/serving/releases](Knative Serving releases page).
```

Comment:
This might be easier to scan if the list was structured as:
____________________________________________________________

For details about the Knative releases, see the following pages:

- [Knative Serving Operator releases](https://github.com/knative/serving-operator/releases)
- etc...